src/app/parcels/ActionBar/ActionModals/DriverOverviewModal.tsx

```diff
@@ -70,15 +70,19 @@ const DriverOverviewModal: React.FC<ActionModalProps> = (props) => {
     const [errorMessage, setErrorMessage] = useState<string | null>(null);
     const [successMessage, setSuccessMessage] = useState<string | null>(null);
 
-    const [driverName, setDriverName] = useState("");
+    const [driverName, setDriverName] = useState<string | null>(null);
     const [date, setDate] = useState(dayjs());
 
     const [isDateValid, setIsDateValid] = useState(true);
 
-    const isInputValid = isDateValid && driverName.length > 0;
+    const isInputValid = isDateValid && driverName;
 
     const onDriverNameChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-        setDriverName(event.target.value);
+        if (event.target.value === "") {
```

Comment:
Referencing the other comment it may make sense to change this to e.g. `event.target.value.trim().length === 0`

Although in an ideal world the name field would probably have some regex to validate that the name has letters in it since now you can still set it as `.` or something - maybe raise a ticket?

Reply:
i'll raise a ticket. it also might be nice to display an error message if the driver name is invalid.